src/tcti-tabrmd.c

```diff
@@ -658,7 +658,7 @@ Tss2_Tcti_Tabrmd_Init (TSS2_TCTI_CONTEXT *context,
 const static TSS2_TCTI_INFO tss2_tcti_info = {
     .name = "tcti-abrmd",
     .description = "TCTI module for communication with tabrmd.",
-    .config_help = "This module takes NO arguments.",
+    .config_help = "This module takes 0 arguments.",
```

Comment:
well, at least the comment is still accurate? 😄

Reply:
Fixed